src/renderer/src/lib.rs

```diff
@@ -12,15 +12,14 @@ extern crate mopa;
 
 extern crate glutin;
 extern crate cgmath;
-
-extern crate amethyst_ecs;
+extern crate specs;
 
 /// Contains the included Render Targets
 pub mod target;
 /// Contains the included Passes
 pub mod pass;
 
-use amethyst_ecs::{Component, VecStorage};
+use self::specs::{Component, VecStorage};
```

Comment:
But do we really need to wrap it? Can't we implement the `Component` trait directly onto the `Light` from inside the engine crate instead of the renderer? It seems like the current method causes a circular dependency, which defeats the purpose of having `amethyst_renderer ` as a separate, modular crate entirely.